Gemfile

```diff
@@ -26,6 +26,9 @@ gem 'search_cop'
 # Pagination
 gem 'kaminari'
 
+# Memoization
+gem 'memoist'
```

Comment:
I used it so we can memoize a RegistryClient inside the Registry model. I can do the memoization manually doe.